Casks/odrive.rb

```diff
@@ -2,14 +2,18 @@
   version "6918"
   sha256 "9107f7634ac70cf017c889738b5bd8e3ac937bd991d5999119832d57407f0c22"
 
-  # downloads can be found at https://www.odrive.com/downloaddesktop
   url "https://d3huse1s6vwzq6.cloudfront.net/odrivesync.#{version}.pkg",
       verified: "d3huse1s6vwzq6.cloudfront.net/"
-  appcast "https://macupdater.net/cgi-bin/check_urls/check_url_redirect.cgi?url=https://www.odrive.com/downloaddesktop?platform=mac"
   name "odrive"
   desc "Tool to make any cloud storage unified, synchronized, shareable, and encrypted"
   homepage "https://www.odrive.com/"
 
+  livecheck do
+    # https://docs.odrive.com/docs/odrive-usage-guide#install-desktop-sync
```

Comment:
```suggestion
  # From: https://docs.odrive.com/docs/odrive-usage-guide#install-desktop-sync
  livecheck do
```

I always put explanatory comments before the `livecheck` block, so it wouldn't hurt to maintain consistency here.